src/main/java/core/basesyntax/db/StorageDao.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageDao {
```

Comment:
I`m not sure this class must be Dao, just Storage

Reply:
Agree, dao is something that interacts with a database and provides some abstraction and useful methods, here you just store database, so it is and actual db not dao